eva_cttv_pipeline/evidence_string_generation/consequence_type.py

```diff
@@ -31,14 +32,28 @@ def process_consequence_type_file_tsv(snp_2_gene_filepath):
 
             process_gene(consequence_type_dict, variant_id, ensembl_gene_id, so_term)
 
-    return consequence_type_dict, one_rs_multiple_genes
+    return consequence_type_dict
+
+
+def process_consequence_type_dataframe(consequences_dataframe):
+    consequence_type_dict = defaultdict(list)
+    for row in consequences_dataframe.itertuples:
+        variant_id = row[0]
+        ensembl_gene_id = row[2]
+        so_term = row[4]
+
+        if pd.isna(ensembl_gene_id):
+            continue
```

Comment:
I think we don't need this check, since the repeat expansion pipeline is already checking for complete rows.